### PR TITLE
add xvfb to testing/dev dependencies

### PR DESCRIPTION
**Pull request by yobleck**
see: #2655

### CODE REVIEW

**Comment by elParaguayo** [comment]
Can you rebase onto latest master? That should fix the test errors.

File: docs/manual/hacking.rst [hunk 1]
```diff
@@ -38,7 +39,7 @@ both backends, specify as arguments to pytest:
     pytest --backend wayland  # Test just Wayland backend
     pytest --backend x11 --backend wayland  # Test both
 
-Testing with the X11 backend requires Xephyr_ in addition to the core
+Testing with the X11 backend requires Xephyr_ and/or xvfb in addition to the core
```
Comment (m-col) [inline docs/manual/hacking.rst:42]: Is this an and/or situation? I thought Xephyr was always needed for X tests.

Comment (yobleck) [reply docs/manual/hacking.rst:42]: you're right. Looks like xvfb just runs xephyr headless

**Comment by ramnes** [comment]
Why closing?

**Comment by yobleck** [comment]
> Why closing?

I had to nuke my fork because it was in a weird state and I didn't how how to revert it.
I'll come back to this later. Sorry for the inconvenience.

**Pull request closed without merging**